src/test/java/uk/gov/hmcts/reform/professionalapi/service/impl/PaymentAccountServiceImplTest.java

```diff
@@ -100,37 +100,38 @@ public void retrievePaymentAccountsByPbaEmailWhenConfigTrue() {
 
         verify(organisationMock, times(1)).setPaymentAccounts(any());
         verify(organisationMock, times(1)).setUsers(any());
-        verify(organisationMock, times(1)).setPaymentAccounts(any());
         verify(professionalUserRepositoryMock, times(1)).findByEmailAddress("some-email");
     }
 
+    @Test(expected = Exception.class)
+    public void testThrowsExceptionWhenEmailInvalid() {
+        when(sut.findPaymentAccountsByEmail("some-email")).thenReturn(organisation);
+    }
+
     @Test
-    public void retrievePaymentAccountsByPbaEmailWhenConfigFalse() {
+    public void editPaymentAccountsByOrganisationIdentifier() {
```

Comment:
prepend test to method name

Reply:
added not sure why it is not reflecting ..